web/modules/custom/portland/modules/portland_zendesk/src/Plugin/WebformHandler/ZendeskHandler.php

```diff
@@ -422,7 +422,7 @@ public function buildConfigurationForm(array $form, FormStateInterface $form_sta
         '#description' => $this->t(
           "<div id=\"help\">
           To set the value of one or more custom fields in the new Zendesk ticket, in <a href=\"https://learn.getgrav.org/16/advanced/yaml#mappings\" target=\"_blank\">YAML format</a>, specify a list of pairs consisting of IDs and values.
-          You may find the custom field ID when viewing the list of <a href=\"https://${zendesk_subdomain}.zendesk.com/agent/admin/ticket_fields\" target=\"_blank\">Ticket Fields</a> in Zendesk, or by clicking <strong>Field Reference</strong>
+          You may find the custom field ID when viewing the list of <a href=\"https://{$zendesk_subdomain}.zendesk.com/agent/admin/ticket_fields\" target=\"_blank\">Ticket Fields</a> in Zendesk, or by clicking <strong>Field Reference</strong>
```

Comment:
Good catch! Did you use a code scanner to find this?

Reply:
No, after updating PHP, Drupal gave me a warning somewhere that the old variable format was deprecated and that we should use the newer format.